raiden/tests/scenarios/bf2_long_running.yaml

```diff
@@ -0,0 +1,332 @@
+# opening channels, depositing, transferring, waiting keeping the raiden node alive for 
+# a long time in the process checking whether all distributed system in raiden network 
+# work accurately with a sufficiently dirty state of blockchain for a long time.
+
+scenario:
+  serial:
+    name: "Long running scenario"
+    tasks:
+      - parallel:
+          name: "Create channels"
+          tasks:
+            - open_channel: {from: 0, to: 1, total_deposit: 0, expected_http_status: 201}
+            - open_channel: {from: 1, to: 2, total_deposit: 0, expected_http_status: 201}
+            - open_channel: {from: 2, to: 3, total_deposit: 0, expected_http_status: 201}
+      - serial:
+          name: "Allow time for nodes to learn about their channels"
+          tasks:
+            - wait: 30
+      - parallel:
+          name: "Verify if channels were opened correctly"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 0, balance: 0,  state: "opened"}
+      - parallel:
+          name: "0 1 2 3 make a deposit of 100 to their channels"
+          tasks:
+            - deposit: {from: 0, to: 1, total_deposit: 100_000_000_000_000_000}
+            - deposit: {from: 1, to: 0, total_deposit: 100_000_000_000_000_000}
+            - deposit: {from: 1, to: 2, total_deposit: 100_000_000_000_000_000}
+            - deposit: {from: 2, to: 1, total_deposit: 100_000_000_000_000_000}
+            - deposit: {from: 2, to: 3, total_deposit: 101_000_000_000_000_000}
+            - deposit: {from: 3, to: 2, total_deposit: 101_000_000_000_000_000}
+      - parallel:
+          name: "Checking if the deposit was successful"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 100_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 100_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 100_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 100_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 101_000_000_000_000_000, balance: 101_000_000_000_000_000, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 101_000_000_000_000_000, balance: 101_000_000_000_000_000, state: "opened"}
+      - parallel:
+          name: "Deposit to a channel that does not exist"
+          tasks:
```

Comment:
Just a note: These are also tested in the api integration tests, but I think it's fine to keep them in for now, for the sake of having a scenario where a lot of stuff is going on.